pipeline/export/nwb.py

```diff
@@ -220,29 +318,42 @@ def datajoint_to_nwb(session_key, raw_ephys=False, raw_video=False):
 
         # ---- Raw Ephys Data ---
         if raw_ephys:
-            ks_dir_relpath = (ephys_ingest.EphysIngest.EphysFile.proj(
-                ..., insertion_number='probe_insertion_number')
-                              & insert_key).fetch('ephys_file')
+            ks_dir_relpath = (
+                ephys_ingest.EphysIngest.EphysFile.proj(
+                    ..., insertion_number="probe_insertion_number"
+                )
+                & insert_key
+            ).fetch1("ephys_file")
             ks_dir = ephys_root_data_dir / ks_dir_relpath
             npx_dir = ks_dir.parent
 
             try:
-                next(npx_dir.glob('*imec*.ap.bin'))
+                next(npx_dir.glob("*imec*.ap.bin"))
             except StopIteration:
-                raise FileNotFoundError(f'No raw ephys file (.ap.bin) found at {npx_dir}')
-
-            sampling_rate = (ephys.ProbeInsertion.RecordingSystemSetup & insert_key).fetch1('sampling_rate')
-            probe_id, probe_type = (ephys.ProbeInsertion & insert_key).fetch1('probe', 'probe_type')
+                warnings.warn(f"No raw ephys file found at {npx_dir}")
```

Comment:
Instead of raising the `FileNotFoundError`, the function will warn the user that no ephys file exists for a particular directory, and continue the loop